azure-pipelines.yml

```diff
@@ -17,17 +11,15 @@ pr:
     - dev
     - in-proc
     - release/4.*
-    - release/inproc6/4.*
-    - release/inproc8/4.*
+    - release/in-proc
```

Comment:
Is this even needed for this branch? The `dev` branch should likely drop all references to the `inproc` branches.

Reply:
@jviau suggested to keep both in sync,

Reply:
Yeah I like to keep the trigger section consistent across all branches to avoid merge conflicts. I _believe_ there is some cases where ADO only examines the default branch for triggers

Reply:
I'm OK keeping this. Just wanted to make sure we were being intentional about keeping this.